### DISCUSSION TITLE
提供一些临时性的帮助作为回报

### DISCUSSION THREAD

**Discussion by xiaohuazi123** [rbind/yihui#1556] [General]
请问临时性的帮助是指代哪些？

**Comment by yihui** [comment]
是两方面：一是赞助（<https://github.com/sponsors/yihui>），二是帮忙介绍工作机会。不过根据目前收到的海量消息，这两方面我已经都不是特别需要了。工作应该是可以找到的，到时候我也会把收到的赞助都捐出去。非常感谢你的关注！
